docs/collecting-application-metrics.md

```diff
@@ -209,3 +220,127 @@ To scrape and forward exposed metrics to Sumo Logic, please follow one of the fo
 
 - [Application metrics are exposed (one endpoint scenario)](#application-metrics-are-exposed-one-endpoint-scenario)
 - [Application metrics are exposed (multiple enpoints scenario)](#application-metrics-are-exposed-multiple-enpoints-scenario)
+
+## Metrics modifications
```

Comment:
Can you also update https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/best-practices.md#excluding-metrics and other metrics sections in the best practices doc with this new info?

Reply:
Should I write more information there or linking to this document will be sufficient?

Reply:
I'd provide some examples. It's structurally not the same thing either, as you may want to filter and modify non-application metrics.